src/ripple/core/impl/DatabaseCon.cpp

```diff
@@ -46,7 +46,9 @@ DatabaseCon::DatabaseCon (
     {
         try
         {
-            session_ << initStrings[i];
+            soci::statement st = session_.prepare <<
+                initStrings[i];
+            st.execute(true);
```

Comment:
Because what was there originally _should_ have worked, but actually had some internal temp sqlite object get an invalid reference count and call `terminate()`. Put another way, errors were not actually ignored. I'm reasonably sure it's a bug in sqllite, but I couldn't find it, so I used a workaround that also worked.